app/shared/services/SchService.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @ngdoc service
+ * @name ipaClientAngularApp.SchService
+ * @description
+ * # SchService
+ * Service in the ipaClientAngularApp.
+ */
+class SchService {
+  constructor() {
+    return {
+      getUnits: function (course) {
```

Comment:
The code looks good,
However if I was trying to find a getUnits helper function, I don't think I'd ever think to look for it in the SchService.

My thinking goes something like:
The concepts in our code (courses, units, SCH, sections, terms, schedules, etc) are all related to each other, and it's up to us to determine if we want to architect something as merely a property of a larger concept, or as its own concept.

My first thought is renaming schService to courseService, as they both are closely connected to courses, and for the moment have very little logic surrounding them so they might not need to be separated.

What do you think?

Reply:
That sounds sensible- I went ahead and renamed it to CourseService.